duckduckgo/Android
# Merge network data

Asana Issue URL: https://app.asana.com/0/488551667048375/471193197990040

## Description
Removes major network counts

## Steps to Test this PR:
1. Open a webpage
1. Open the Privacy Dashboard
1. Note that there is now only one networks entry
